### ISSUE TITLE
Future of Gantry

### ISSUE THREAD

**Issue by Schnok** [gantry/gantry5#3182]
Hello everyone,

Like many I suppose, I see that there is less reaction from the Gantry team here, and especially from Rocket Theme on their forum. There are many requests that have no response.

So I'm worried because all the sites I've built over the last few years have been based on Gantry, and I still have some that I need to move to Joomla 4.

It would be great if someone from the Team could come here and give us some news, even (or especially) if it's bad news...

Thanks in advance.

**Comment by thisisscotts** [comment]
Agreed - I have built many websites using Gantry (with booth Joomla and Wordpress, but mostly with the latter) and would like to continue doing so.

Otherwise, if there is a similar (or otherwise easy-to-use) framework that others are using, I'd be keen to hear about it.

**Comment by N8Solutions** [comment]
Here are 2 threads, one on the Rocket Theme forum and the other one on Inspire Theme's discussing this. They are worth a read.

https://rockettheme.com/forum/general-discussion/290182-solved-will-future-support-continue

https://www.inspiretheme.com/forums/what-does-the-future-look-like

**Comment by Schnok** [comment]
@N8Solutions Thanks for the links, they're interesting.

**Comment by Schnok** [comment]
@hexplor Thank you for your reply. 
It's very reassuring because I couldn't see myself looking for any solution other than Gantry / RT.
I'm going to take part in the tests.
Best regards

**Comment by anildagia** [comment]
Joomla has released Joomla 5. Any plans for Joomla 5 compatibility?

**Comment by gioras** [comment]
@hexplor like @Schnok I wanted to take a moment to convey my sincere appreciation and gratitude for your relentless efforts in continuing the development of Gantry 5 for Joomla 5.

Best Regards,

--Giora

**Comment by Seahawk59** [comment]
I was struggling with the Gantry 5.5.17 update on the Joomla 4.4 site to update to J5. At the end of the update, it showed a 500 error. I restored and switched off several plugins and retried again and then got the following error at the end of the installation:

Failed to load 'g5_hydrogen' template: Please install Gantry 5 Framework!
An error has occurred.
0 Class "JPlugin" not found 

This happened when I tested another plugin where the Joomla Compatibility plugin was disabled in Joomla 5.0.1. I then had to go to CPanel to switch on the Joomla Behaviour - Backward Compatibility plugin in the PHPAdmin. Thereafter the Template worked as expected. So it means that something is bugging the template without Compatibility Extention activated.
